src/Compilers/CSharp/Portable/Symbols/PropertySymbolExtensions.cs

```diff
@@ -63,7 +63,7 @@ public static bool HasRefOrOutParameter(this PropertySymbol property)
         {
             foreach (ParameterSymbol param in property.Parameters)
             {
-                if (param.RefKind != RefKind.None)
+                if (param.RefKind == RefKind.Ref || param.RefKind == RefKind.Out)
```

Comment:
Adding `IsConstModReqIsConsumedInRefCustomModifiersPosition_Indexers_ReturnTypes`. This change basically allows to pass ref-readonly parameters to indexer methods.

---
In reply to: [118061963](https://github.com/dotnet/roslyn/pull/19658#discussion_r118061963) [](ancestors = 118061963)